src/main/io/serial_cli.c

```diff
@@ -2516,7 +2516,9 @@ static void cliDumpProfile(uint8_t profileIndex, uint8_t dumpMask, master_t *def
     changeProfile(profileIndex);
     cliPrint("\r\n# profile\r\n");
     cliProfile("");
+    cliPrint("\r\n");
     dumpValues(PROFILE_VALUE, dumpMask, defaultConfig);
+    cliRateProfile("");
```

Comment:
Because the rate profile is stored as part of the profile: 

https://github.com/mikeller/betaflight/blob/2d1a1865ded958b626c8157488b60595958a32d3/src/main/config/config_profile.h#L22

And it's the (current) rateprofile that is stored with the profile being dumped, since we are switching profiles for the dump.